.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
```

Comment:
Maybe lets run pre-commit autoupgrade?  I think the latest version of this is something around `v3.x`